### PR TITLE
Deprecate shared scripts

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

Deprecates shared scripts

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recmZk1fbT1H98PLn?blocks=bipIkdtYHmboUfJut

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Ran ETL locally

## Does any infrastructure work need to be done before this PR can be pushed to production?

N/A

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1 @@
-[submodule "shared_scripts"]
-	path = shared_scripts
-	url = https://github.com/serotracker/shared_scripts.git
+
```
Comment (rahularoradfs) [inline .gitmodules:1]: can just remove this file

File: README.md [hunk 2]
```diff
@@ -6,13 +6,7 @@ Server side code for the International Immunity Tracker
...
-
-Note that in the parent repo, the reference to the submodule is NOT to the submodule git repo as a whole in its constantly-updating form. Rather, it is to a specific commit. You need to deliberately integrate the updated version of submodule code. To do so, use `git submodule update --remote`, and then commit that change. Alternatively, `cd shared_scripts; git pull` then commit that change.
-
-Submodules enable you to update submodule code at the same time as you update core code. The easiest way to do this is probably to clone the submodule's git repo into another repo (outside of the parent module), modify it there, and pull in the the updated version of the submodule to the parent repo per above. Alternatively, you can check out a branch in the submodule (`cd shared_scripts; git checkout main`). After modifying the submodule as desired, you can push these changes, either from the submodule (`git push`) or the parent repo (`git push --recurse-submodules=on-demand`).
+`git clone https://github.com/serotracker/iit-backend.git`
```
Comment (rahularoradfs) [inline README.md:9]: booo use ssh not html

**Review by abeljoseph — COMMENTED** [review]
LGTM after making the changes Rahul requested

**Review by abeljoseph — APPROVED** [review]
LGTM

**Pull request merged**